types/api/apps.d.ts

```diff
@@ -23,15 +23,15 @@ type AppsState = {
 };
 
 type AppBinding = {
-    app_id: string;
-    location: string;
+    app_id?: string;
```

Comment:
Is that field really optional?

Reply:
Yes and no. The thing is... many of these fields can be inherited. So... the "root binding" should have them, but the rest don't need to define it. The `cleanBindings` function will take care of that.

I thought about making the distinction between clean and raw bindings, but it complicated many things, so I preferred to go with just making them optional everywhere.

Reply:
Should we make the distinction clear with a comment?